### DISCUSSION TITLE
When Safari vertical scrollbar appears the breakpoint changes

### DISCUSSION THREAD

**Discussion by petros** [tailwindlabs/tailwindcss#1521] [General]
Has anyone ever had to deal with such a problem?

I am not sure if this is expected or a bug or just an improvement opportunity with Tailwind CSS on Safari.

- My display resolution is 1280x800 (MacBook Pro)
- Using latest Safari
- The main content is not long enough to cause Safari's vertical scrollbar to appear
- Clicking somewhere in the page expands the content vertically and the scrollbar appears
- Immediately the breakpoint changes which causes the content to change its left/right margins 

This does not happen on Chrome.

Is this a Safari issue? Could it be because of the way I have set up my main content with Tailwind CSS?

![demo](https://user-images.githubusercontent.com/28818/78289428-dd4b5780-752a-11ea-8d60-bdd3b631131a.gif)

**Comment by tlgreg** [comment]
Yeah. Webkit implemented media queries this way but the spec does say that the device width should include the scroll bar too, exactly because of this reason. Chrome fixed it for quite some time ago.

**Reply by petros** [reply]
Thanks so much @tlgreg for confirming this is an issue with Safari. Judging from how fast Apple solves problems, I am not holding my breath. Would you suggest any workaround?

**Comment by egiv** [comment]
Hi. I had some issue but only in iPad mini responsive design mode. Searching any workaround ways too.
